k8s/aat/common/ia/case-documents-api.yaml

```diff
@@ -14,24 +14,14 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: ia-case-documents-api
-    version: 0.0.2
+    version: 0.0.4
   values:
     java:
       replicas: 2
-      memoryRequests: '768Mi'
-      cpuRequests: '500m'
-      memoryLimits: '2048Mi'
-      cpuLimits: '1000m'
       useInterpodAntiAffinity: true
-      applicationInsightsInstrumentKey: "bf7eeffe-bd72-4350-b336-cf4020806c02"
       image: hmctspublic.azurecr.io/ia/case-documents-api:prod-4fc6dc5a
       ingressHost: ia-case-documents-api-aat.service.core-compute-aat.internal
-      environment:
-        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
-          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.4.1.jar
-      aadIdentityName: ia
```

Comment:
I found that we do not have `aadIdentityName: ia` in some yaml configs, so I removed from the others too? Is it ok @timja ?